packages/callout-block/src/helpers/getTextColor.ts

```diff
@@ -1,10 +1,14 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { getReadableColor, isDark } from '@frontify/guideline-blocks-shared';
+import { getReadableColor, toShortRgba } from '@frontify/guideline-blocks-shared';
+import tinycolor from 'tinycolor2';
 import { Appearance } from '../types';
 
 export const getTextColor = (appearance: Appearance, color: string, backgroundColor: string): string => {
-    const isDarkColor = isDark(color);
+    const parsedColor = tinycolor(toShortRgba(color));
+    const brightness = parsedColor.getBrightness();
+    const isDarkColor = brightness < 150;
```

Comment:
Can this logic be in the 'shared' folder, so we don't need to add tinycolor to the callout dependencies?

Reply:
Refactored it to have a threshold option on the `isDark` utility 👍🏻